packages/pyolite-kernel/src/worker.ts

```diff
@@ -216,6 +216,17 @@ function complete(content: any) {
   return results;
 }
 
+/**
+ * Inspect the code submitted by a user.
+ *
+ * @param content The incoming message with the code to inspect.
+ */
+function inspect(content: any) {
```

Comment:
Looks like we could use a stronger type than `any` here?

And add `content` to the docstring above.

Reply:
`content` is already in docstring. I have now added type definition.